MNE/rtProcessing/ttt.h

```diff
@@ -0,0 +1,11 @@
+#ifndef TTT_H
+#define TTT_H
+
+
+class ttt
+{
+public:
+    ttt();
+};
+
+#endif // TTT_H
```

Comment:
I did not add this class. My task is not related to rtProcessing.

>> @@ -0,0 +1,11 @@
>> +#ifndef TTT_H
>> +#define TTT_H
>> +
>> +
>> +class ttt
>> +{
>> +public:
>> +    ttt();
>> +};
>> +
>> +#endif // TTT_H
>
> What kind of class is this? is it getting used somewhere? If not, please
> delete and recommit. If you need new helper class in the the rtProcessing
> library, please put them into MNE/rtProcessing/helpers  folder (non
> existent yet, you would need to create it). cheers, lorenz
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/mne-tools/mne-cpp/pull/205/files/f411a13736b94c4a6f3f0fd2794c04d2fccb9047#r67705760



The information in this e-mail is intended only for the person to whom it is
addressed. If you believe this e-mail was sent to you in error and the e-mail
contains patient information, please contact the Partners Compliance HelpLine at
http://www.partners.org/complianceline . If the e-mail was sent to you in error
but does not contain patient information, please contact the sender and properly
dispose of the e-mail.